Expensify/App
# Added tooltips to all Icon buttons

@iwiznia Can you please review this?

### Details
Wrapped all TouchableOpacity + Icon components with Tooltip.

### Fixed Issues

$ #4499 

### Tests
1. Verified tooltips for IOUModal, Search, ReportActionCompose, AttachmentView, VideoChat.
2. Verified Tooltips for Web and Desktop
3. Tested for no impact on Mobile Web, iOS and Android



### QA Steps
1. Need to verify if all the icon buttons are covered. (exclude Green FAB)
2. Need to verify if any tooltip has any alignment issue.
3. Added tooltips for icon buttons in  IOUModal, Search, ReportActionCompose, AttachmentView, VideoChat.



### Tested On


### Screenshots


#### Web


<img width="165" alt="tooltip-icon-close-download png" src="https://user-images.githubusercontent.com/3069065/129442148-f7125e17-4960-407b-9710-f984289250ab.png">
<img width="122" alt="tooltip-icon-close-web" src="https://user-images.githubusercontent.com/3069065/129442151-dfaa2a42-0075-4e58-bdaa-fcb86061baed.png">
<img width="203" alt="tooltip-icon-emoji-web" src="https://user-images.githubusercontent.com/3069065/129442152-bd2aac77-a50d-4d15-94d7-92e964e27fee.png">
<img width="128" alt="tooltip-icon-pin-web" src="https://user-images.githubusercontent.com/3069065/129442153-1de688a7-a63c-446d-a5a6-d54cc82dc57b.png">
<img width="369" alt="tooltip-icon-search-web" src="https://user-images.githubusercontent.com/3069065/129442154-73edebdb-04bb-4ade-bd9e-489d42dce658.png">
<img width="104" alt="tooltip-icon-unpin-web" src="https://user-images.githubusercontent.com/3069065/129442156-e65a7bb8-45e3-47a3-9f86-d7515e55aedc.png">
<img width="187" alt="tooltip-icon-vc-web" src="https://user-images.githubusercontent.com/3069065/129442157-9164a16f-9a1f-4c81-aae7-4dd9c14fe348.png">


#### Mobile Web
NA


#### Desktop



<img width="125" alt="tooltip-icon-close-desktop" src="https://user-images.githubusercontent.com/3069065/129442286-960ec1e9-08a7-4643-8536-e788a7e60170.png">
<img width="289" alt="tooltip-icon-download-desktop" src="https://user-images.githubusercontent.com/3069065/129442288-e8c1278c-f08a-4f07-bb62-95052ea44dd4.png">
<img width="270" alt="tooltip-icon-emoji-desktop" src="https://user-images.githubusercontent.com/3069065/129442289-f656a7ea-4085-4641-90d5-648488df3373.png">
<img width="183" alt="tooltip-icon-pin-desktop" src="https://user-images.githubusercontent.com/3069065/129442290-0b3fadb6-eb91-41ec-a742-77a0857b9977.png">
<img width="426" alt="tooltip-icon-search-desktop" src="https://user-images.githubusercontent.com/3069065/129442291-43231615-d608-4748-adf6-49f36dfeec7c.png">
<img width="298" alt="tooltip-icon-unpin-desktop" src="https://user-images.githubusercontent.com/3069065/129442294-e108b36e-15f3-4e67-88fd-1c165525ee5f.png">
<img width="170" alt="tooltip-icon-vc-desktop" src="https://user-images.githubusercontent.com/3069065/129442295-a8055129-c392-4320-a7fc-98ddbd333151.png">


#### iOS
NA


#### Android
NA
